modules/gcp_nodepool/gcp-nodepool.yaml

```diff
@@ -70,6 +70,11 @@ inputs:
     validator: list(include('taint'), required=False)
     description: Taints to add to the nodes in this nodegroup.
     default: []
+  - name: guest_accelerators
+    user_facing: true
+    validator: list(include('guest_accelerator'), required=False)
+    description: GPU accelerators to add to the nodes in this nodegroup.
```

Comment:
GCP supports this, and there is a possibility, however slim, that customers might want this. We can always constrain our module API, but this introduces the risk of additional work if the need ever arises.

Reply:
Hmm: failing with:
```
│ Error: error creating NodePool: googleapi: Error 400: All GPUs in a node-pool should be of the same GPU type., badRequest
│
│   with module.nodegroup2.google_container_node_pool.node_pool,
│   on ../../modules/gcp_nodepool/tf_module/main.tf line 55, in resource "google_container_node_pool" "node_pool":
│   55: resource "google_container_node_pool" "node_pool" {
│
```
which suggests that it would be pointless to even support multiple guest accelerator types.

Reply:
Simplified to a single guest accelerator type per node group.